### ISSUE TITLE
Compatibility issue between VR runtimes

### ISSUE THREAD

**Issue by cabanier** [immersive-web/webxr-hand-input#113]
While porting the Oculus browser to OpenXR, we realized that there is a difference between some joints. 
This manifests itself as a hand that is slightly deformed.
Wondering which runtime is correct, I asked @Yonet how hands are rendering on the Hololens. It turns out that they are [rendering deformed](https://www.youtube.com/watch?v=d_VFaVi4ZXE) as well.

Is this the correct forum to solve this?

I've been thinking about how we can fix it and I believe that the cleanest way would be to return a new profile name that points to an updated hand mesh. This mesh would be compatible with future Oculus browser and current Hololens. Hololens would have to update its native code to return this new profile name.
Alternatively, I could update the profile name of the current Oculus browser and we copy the existing profile to it.

@toji @Manishearth @thetuvix, what do you think?

**Comment by cabanier** [comment]
Maybe I'm overcomplicating it and I should just fix Oculus browser.
Then when we ship, we update the profile...

**Comment by AdaRoseCannon** [comment]
Yeah, fixing it and keeping the profile name makes more sense to me since
afaik the only headsets that support it are those two

**Comment by cabanier** [comment]
@toji I believe you updated the profile in the past. If you have a browser that returns different joints, would you be able to fix it up?
